CHANGELOG.md

```diff
@@ -10,17 +27,16 @@ App Center Auth is [retired](https://aka.ms/MBaaS-retirement-blog-post) and has
 
 App Center Data is [retired](https://aka.ms/MBaaS-retirement-blog-post) and has been removed from the SDK.
 
-___
+### App Center Crashes
 
-## Version 2.6.2 (Under development)
+#### Android
 
-### App Center
+* **[Fix]** Fix incorrect app version when an NDK crash is sent after updating the app.
+* **[Behavior change]** Change the path to the minidump directory to use a subfolder in which the current contextual data (device information, etc.) is saved along with the .dmp file.
 
 #### iOS
 
-* **[Fix]** Fix an error where **appcenter.podspec.json** could not be found when using CocoaPods version 1.8.x.
-* **[Fix]** Fix issues with `use_frameworks!` directive.
-* **[Fix]** Fix an issue where React Native SDK would not send wrapperSdk information.
+* **[Improvement]** Update PLCrashReporter to 1.4.0.
 
 ___
 
```

Comment:
Please help me check if the content of CHANGELOG.ms is correct?

Reply:
Looks correct